api/hypershift/v1alpha1/hostedcluster_types.go

```diff
@@ -1573,7 +1573,6 @@ type AzurePlatformSpec struct {
 	VnetID            string `json:"vnetID"`
 	SubnetID          string `json:"subnetID"`
 	SubscriptionID    string `json:"subscriptionID"`
-	MachineIdentityID string `json:"machineIdentityID"`
```

Comment:
So, machine identity is not needed at all during HostedCluster creation?

Reply:
No